clio-server/src/main/scala/org/broadinstitute/clio/server/service/PersistenceService.scala

```diff
@@ -30,14 +31,14 @@ class PersistenceService private (persistenceDAO: PersistenceDAO,
     * @tparam TK The type of the Transfer Key DTO.
     * @tparam TM The type of the Transfer Metadata DTO.
     * @tparam D  The type of the Document.
-    * @return A future result of the upsert.
+    * @return A future result of the ID for this upsert.
```

Comment:
How about "a future ID for this upsert" or just "an ID for this upsert" ?

Reply:
ok